functions.php

```diff
@@ -0,0 +1,15 @@
+<?php
+function generateCharacterHtml(array $allCharacters): string {
```

Comment:
this name implies the function only generates the html for one character rather than all characters - should be changed to be more semantic